apps/recnet-api/src/database/repository/user.repository.ts

```diff
@@ -107,14 +107,22 @@ export default class UserRepository {
         select: user.select,
       });
 
-      await prisma.inviteCode.update({
+      const inviteCode = await prisma.inviteCode.update({
         where: { code: createUserInput.inviteCode },
         data: {
           usedById: userInTransaction.id,
           usedAt: new Date(),
         },
       });
 
+      // follow the person who gave the invite code
+      await this.prisma.followingRecord.create({
```

Comment:
```suggestion
      await prisma.followingRecord.create({
```

You should use `prisma` instance in transaction mode, otherwise it will result in errors.

Reply:
good catch